src/components/Book.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import PropTypes from 'prop-types';
 
```

Comment:
- [ ] Based on the project's requirement, the redux `removeBook` action creator should be dispatched in this component when the _**Remove**_ button is clicked. I would advise you to pass each Book's **_ID_** as a prop from the parent component `BookList` and pass the **_ID_** as a payload when dispatching the `removeBook` action creator whenever the Remove button is clicked 💻

Reply:
fix the removeBook function thanks for noticing